cowprotocol/accounting/slippage/slippage_prices_4064601.sql

```diff
@@ -25,14 +26,24 @@ with token_times as (
     group by 1, 2
 ),
 
--- Precise prices are prices from the Dune price feed.
-precise_prices as (
+-- Fetching all additional price feeds that are synced to Dune
+imported_price_feeds as (
+    select
+        a.hour,
+        a.token_address,
+        a.decimals,
+        a.price_unit
+    from "query_4252674" as a inner join token_times as tt on a.hour = tt.hour and a.token_address = tt.token_address
+    where a.price_unit < 10000000 -- here we filter all tokens with price more than $1M, as these are probably bogus prices
+),
+
+-- the Dune price feed; note that this is computed on Dune and is not part of the imported_price_feeds_raw table.
+dune_price_feed as (
     select -- noqa: ST06
         date_trunc('hour', minute) as hour, --noqa: RF04
         token_address,
         decimals,
-        avg(price) as price_unit,
-        avg(price) / pow(10, decimals) as price_atom
```

Comment:
Is it intentional that `price_atom` is removed here?

Reply:
The reason was that we can work with units and at the very end introduce the price of an atom as we know the number of decimals. This was to avoid repetition of caclulations (e.g., computing the median for the price_unit and price_atom)